pkg/cmd/pulumi/packagecmd/package_info_test.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2025, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package packagecmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/pulumi/pulumi/pkg/v3/codegen/schema"
+	"github.com/stretchr/testify/require"
+)
+
+func generateSchema(t *testing.T) []byte {
+	spec := &schema.PackageSpec{
+		Name:    "test",
+		Version: "0.0.1",
+		Description: `test description
+markdown formatted
+
+another paragraph`,
+		Resources: map[string]schema.ResourceSpec{
+			"test:index:Test": {
+				ObjectTypeSpec: schema.ObjectTypeSpec{
+					Description: "test\nresource\ndescription\n\nanother paragraph",
+					Properties: map[string]schema.PropertySpec{
+						"prop1": {
+							Description: "this is a string property",
+							TypeSpec: schema.TypeSpec{
+								Type: "string",
+							},
+						},
+						"arrayProp": {
+							Description: "this is an array property",
+							TypeSpec: schema.TypeSpec{
+								Type: "array",
+								Items: &schema.TypeSpec{
+									Ref: "#/types/test:index:TestType",
+								},
+							},
+						},
+						"enumProp": {
+							Description: "this is an enum property",
+							TypeSpec: schema.TypeSpec{
+								Ref: "#/types/test:index:EnumType",
```

Comment:
This should have Type: "ref" I think

Reply:
I don't think so.  It is documented as being ignored in the schema: https://github.com/pulumi/pulumi/blob/tg%2Fpackage-info/pkg/codegen/schema/pulumi.json#L302.  

From a quick look at a couple of schemas (aws/awsx) don't have the type set at all when there's a ref specified.